### PR TITLE
Add python and golang linter CI workflows

### PR DESCRIPTION
**Pull request by akiselev1**
This PR enables a complete linter coverage for python and selected checks for golang.

### CODE REVIEW

File: testpmd/cmd/pci.go [hunk 1]
```diff
@@ -2,7 +2,6 @@ package main
 
 import (
 	"fmt"
-	"io/ioutil"
```
Comment (jianzzha) [inline testpmd/cmd/pci.go:5]: Is this changed generated by the linter?

**Review by jianzzha — COMMENTED** [review]
I think it would be a good idea to also include two local script for the same linter check, one for python and one for golang. This way the users can do the local check up before PR is submitted. Also put the this local check into a contribution doc. This is the lesson I learnt by seeing how Intel struggled on the linter checking in the sr-iov-test repository

**Comment by akiselev1** [comment]
I added CI documentation with linter call examples in the form of CONTRIBUTING.md. That was largely derived from SR-IOV project (which I think is nearly perfect) with some changes, relevant to netgauge.

**Review by jianzzha — APPROVED** [review]
Looks good

**Review by dkosteck — APPROVED** [review]
Looks good to me

**Pull request merged**